.github/workflows/build-all-distros-nightly.yaml

```diff
@@ -33,3 +33,21 @@ jobs:
       - name: build all
         run: |
           make build-all
+      - name: slack on success
+        if: success()
+        uses: actions-ecosystem/action-slack-notifier@fc778468d09c43a6f4d1b8cccaca59766656996a
+        with:
+          slack_token: ${{ secrets.WEAVEWORKS_SLACK_EKSCTLBOT_TOKEN }}
+          message: ":cool-doge: build-all-distros successful :greentick: "
+          channel: team-pitch-black
+          color: green
+          verbose: false
```

Comment:
It would be quite a lot of noise to have this notification every day after this runs at 9am 🤔 We can remove this success notification.
```suggestion
```